bower.json

```diff
@@ -14,7 +14,9 @@
     "toastr": "^2.1.2",
     "font-awesome": "~4.5.0",
     "typeahead.js": "^0.11.1",
-    "jquery.tagsinput": "^1.3.6"
+    "jquery.tagsinput": "^1.3.6",
+    "loaders.css": "^0.1.2",
+    "markdown-it": "^8.3.1"
```

Comment:
If you are not planning on using the `ember-markdown-it` addon, I would remove it from `package.json`.  Then, you are going to need a few things to make markdown-it work - (this is how I got it working - there may be a better way).

1) Add `app.import(path.join(app.bowerDirectory, 'markdown-it/dist/markdown-it.js'));` to `ember-cli-build.js`

2) You'll now see something like this: `Could not find module: markdown-it`.  Remove `import markdownit from 'markdown-it';` from `addon/helpers/markdown-viewer.js`.   It looks like markdownit is available as a global.

3) jshint ignore:line where markdown it is used in `addon/helpers/markdown-viewer.js`